vault/auth_external_plugin_test.go

```diff
@@ -1,318 +0,0 @@
-package vault
```

Comment:
+1 for the rename and repurpose of this code. Would be nice to see the changes pre-rename. I pulled the branch down, did a reset and side-by-side diff'ed. Everything looks great!

If there was a separate commit before moving the file, it'd make that a bit easier, but not a huge deal, I'm just lazy :).

Reply:
Yeah sorry about that, I should have split the commits